pipeline-runner/renv/settings.dcf

```diff
@@ -1,8 +1,10 @@
+bioconductor.version:
 external.libraries:
-ignored.packages:
+ignored.packages: devtools, roxygen2, usethis, languageserver, styler
 package.dependency.fields: Imports, Depends, LinkingTo
 r.version:
 snapshot.type: implicit
 use.cache: TRUE
+vcs.ignore.cellar: TRUE
```

Comment:
What's this?

Reply:
The package cellar is a way to provide renv with the source code of packages which may not be available elsewhere (not in cran, bioconductor, github, etc). We don't use this. And it's [default value is true](https://github.com/rstudio/renv/blob/f01698e541237cbf3f59ad041512de66f67b0e11/R/settings.R#L446). I figure that when adding the ignered dependencies the renv function checks for all parameters and adds those which are missing.

The same with bioconductor.version and r.version. which are NULL by default.